test/TestProjects/MediaTypes-Cadl/Generated/MultipleMediaTypesClient.cs

```diff
@@ -0,0 +1,347 @@
+        /// <include file="Docs/MultipleMediaTypesClient.xml" path="doc/members/member[@name='OneBinaryBodyTwoContentTypes(RequestContent,ContentType,RequestContext)']/*" />
+        public virtual Response OneBinaryBodyTwoContentTypes(RequestContent content, ContentType contentType, RequestContext context = null)
+        {
+            Argument.AssertNotNull(content, nameof(content));
+
+            using var scope = ClientDiagnostics.CreateScope("MultipleMediaTypesClient.OneBinaryBodyTwoContentTypes");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateOneBinaryBodyTwoContentTypesRequest(content, contentType, context);
+                return _pipeline.ProcessMessage(message, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <param name="body"> The String to use. </param>
+        /// <param name="contentType"> The Union to use. Allowed values: &quot;application/json&quot; | &quot;application/octet-stream&quot; | &quot;text/plain&quot;. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="body"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="body"/> is an empty string, and was expected to be non-empty. </exception>
+        public virtual async Task<Response> OneStringBodyThreeContentTypesAsync(string body, ContentType contentType, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(body, nameof(body));
+
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Response response = await OneStringBodyThreeContentTypesAsync(body, contentType, context).ConfigureAwait(false);
+            return response;
+        }
+
+        /// <param name="body"> The String to use. </param>
+        /// <param name="contentType"> The Union to use. Allowed values: &quot;application/json&quot; | &quot;application/octet-stream&quot; | &quot;text/plain&quot;. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="body"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="body"/> is an empty string, and was expected to be non-empty. </exception>
+        public virtual Response OneStringBodyThreeContentTypes(string body, ContentType contentType, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(body, nameof(body));
+
+            RequestContext context = FromCancellationToken(cancellationToken);
+            Response response = OneStringBodyThreeContentTypes(body, contentType, context);
```

Comment:
Can `string` implicitly cast to `RequestContent`? I know there is a function `public static RequestContent Create(string content) `

Reply:
Yes. `string` and `BinaryData` can both implicitly cast to `RequestContent`